pkg/nsm/apis/nsmconnect/nsmconnect.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+package nsmconnect;
+
+message Label {
+    map<string ,string> selector = 1;
+}
+
+message Metadata {
+    string Name = 1;
+    string Namespace = 2;
+    Label Labels = 3;
+}
+
+enum InterfaceType {
+    KernelInterface = 0;
+    VhostUser = 1;
+    Memif = 2;
+    Other = 3;
```

Comment:
I'm not sure what one would do with 'Other'.  The NSM has to be able to at least try to give the user the interface type they are asking for, I'm not sure its possible to do that for 'Other' :)

Reply:
Will remove it, I used what you had on the slide as a baseline.